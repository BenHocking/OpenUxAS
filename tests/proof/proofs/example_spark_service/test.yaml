filenames: ["uxas-comms-lmcp_net_client-service-example_spark_service-spark.ads"]
