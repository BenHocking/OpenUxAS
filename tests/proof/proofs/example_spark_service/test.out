uxas-comms-lmcp_net_client-service-example_spark_service-spark.adb:22:34: info: tag check proved
uxas-comms-lmcp_net_client-service-example_spark_service-spark.adb:24:81: info: range check proved
uxas-comms-lmcp_net_client-service-example_spark_service-spark.adb:25:35: info: tag check proved
uxas-comms-lmcp_net_client-service-example_spark_service-spark.adb:25:70: info: range check proved
uxas-comms-lmcp_net_client-service-example_spark_service-spark.ads:33:07: info: initialization of "Recognized_Id" proved
uxas-comms-lmcp_net_client-service-example_spark_service-spark.ads:37:08: info: postcondition proved
uxas-comms-lmcp_net_client-service-example_spark_service-spark.ads:38:36: info: tag check proved
